src/features/chains/metadata.ts

```diff
@@ -12,9 +12,13 @@ import {
 } from '@hyperlane-xyz/sdk';
 import { ProtocolType } from '@hyperlane-xyz/utils';
 
-import { chains } from '../../consts/chains';
+import { chains as ChainsTS } from '../../consts/chains';
+import ChainsJson from '../../consts/chains.json';
```

Comment:
Do we even have a thing that outputs json chain metadata?

Reply:
The CLI commands can output either. There's a `format` flag that can be json or yaml.